test/withStyles_test.jsx

```diff
@@ -341,7 +340,7 @@ describe('withStyles()', () => {
       }))(MyComponent);
 
       // copied
-      const expectedPropTypes = deepmerge({}, MyComponent.propTypes);
+      const expectedPropTypes = Object.assign({}, MyComponent.propTypes);
```

Comment:
this can probably use object spread?

Reply:
Since we run our tests on Node 4, I don't think so. But I suppose I could turn that off, but doesn't seem like it really matters much either way.

Reply:
i think we're using babel-register for our tests tho?

Reply:
i'm relatively sure spread will work here, since tests use `import` and that doesn't work on any node.

Reply:
Ah good point. ok I'll change it